File: quasar/Quasar.DataTransfer/Produtos/Responses/EspecificacaoResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Quasar.DataTransfer.Produtos.Responses
{
    public class EspecificacaoResponse
    {
        public int Codigo { get; set; }
        public string Posicao { get; set; }
        public string Cor { get; set; }
        public DateTime Ano { get; set; }
        public string Veiculo { get; set; }
    }
}

File: quasar-front/src/app/shared/models/responses/especificacao.response.ts
export class EspecificacaoResponse {
  public codigo: number;
  public posicao: string;
  public cor: string;
  public ano: string;
  public veiculo: string;

  constructor(params: Partial<EspecificacaoResponse>) {
    this.codigo = params.codigo || 0;
    this.posicao = params.posicao || '';
    this.cor = params.cor || '';
    this.ano = params.ano || '';
    this.veiculo = params.veiculo || '';
  }
}
